packages/typespec-ts/test/integration/generated/type/union/src/responses.ts

```diff
@@ -26,7 +26,7 @@ export interface StringsOnlySend204Response extends HttpResponse {
 /** The request has succeeded. */
 export interface StringExtensibleGet200Response extends HttpResponse {
   status: "200";
-  body: { prop: string | "b" | "c" };
+  body: { prop: "b" | "c" };
```

Comment:
@joheredi for anonymous non exhaustive enum, what do you think should be our behavior here? I feel like either keep the current behavior or just generate the string ?

Reply:
I think we should scope these changes to Modular only. I would prefer if we keep RLC behavior as it. One thing we may want to update in RLC is to generate `string` instead of `"a" | "b" | string` but not a high priority item

Reply:
Updated to string.